<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workflowBuilder</class>
 <widget class="QDialog" name="workflowBuilder">
  <property name="windowTitle">
   <string>Workflow Builder 0.01</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="GraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QDockWidget" name="dockWidget">
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>524287</height>
        </size>
       </property>
       <widget class="QWidget" name="dockWidgetContents">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="item">
           <property name="text">
            <string>Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="ioput">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>252</width>
              <height>356</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Input/Output</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Input</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QTreeView" name="treeViewIn">
                   <property name="headerHidden">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Output</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QTreeView" name="treeViewOut">
                   <property name="headerHidden">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="description">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>252</width>
              <height>356</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Description</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTextBrowser" name="textEditDesc"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
